src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/AdminSweeper.scala

```diff
@@ -31,54 +38,89 @@ object AdminSweeper {
       .map(_.getEmail)
       .toList
   }
-}
 
-/**
- * Poll for an updated list of admins, and update our user table to reflect this list.
- * Intended to be run via a scheduler from the parent actor
- * TODO- Implement bulk transactions for better scalability. Currently runs a DB transaction for each user whose admin status needs changing.
- */
-class AdminSweeper(pollAdmins: () => List[String], permissionsDataSource: PermissionsDataSource) extends Actor {
+  def apply(pollAdmins: () => List[String],
+            permissionsDataSource: PermissionsDataSource,
+            initialDelay: FiniteDuration,
+            fixedRate: FiniteDuration): Behavior[Command] = {
+    Behaviors.setup { _ =>
+      Behaviors.withTimers { timers =>
+        timers.startSingleTimer(SweepSingleTimer, Sweep, initialDelay)
+        timers.startTimerAtFixedRate(SweepRateTimer, Sweep, fixedRate)
```

Comment:
see comment on `TypedHealthMonitor`

Reply:
Just a note that `TypedHealthMonitor` (and the underlying workbench-util `HealthMonitor`) behave slightly differently than  `AdminSweeper`. `AdminSweeper` can receive the same message `Sweep` repeatedly, and if it's already in a `Sweeping` state it will ignore the incoming request.

FYI- I originally replaced the deprecated `Scheduler.schedule` with   `Scheduler.startTimerAtFixedRate` based on the javadoc comment:

> This has the same semantics as scheduleAtFixedRate

But I concur this is probably better behavior, and we'll do some upgrades while we're here tinkering. ⬆️